src/eb.ts

```diff
@@ -214,6 +214,32 @@ const completionSpec: Fig.Spec = {
         description: "The name of the environment to print",
       },
     },
+    {
+      name: "use",
+      description: "Sets the specified environment as the default environment",
+      args: {
+        name: "environment-name",
+        description: "The name of the environment to use",
+      },
+      options: [
+        {
+          name: ["-r", "--region"],
+          description: "Change the region in which you create environments",
+          args: {
+            name: "region",
+            description: "The region to use",
+          },
+        },
+        {
+          name: "--source codecommit/",
```

Comment:
This appears to be wrong:
```suggestion
          name: "--source",
```